src/mocks/CatalogueItems.json

```diff
@@ -27,6 +27,7 @@
     "days_to_rework": null,
     "drawing_number": null,
     "drawing_link": null,
+    "expected_lifetime": null,
```

Comment:
Should at least one of these have a value to display in a test? Would slightly improve coverage if you use one already displayed in one of the unit tests.

Reply:
Yeah, that's a good idea, I'll add at least 1 so that it should show up in the Catalogue Items Table unit tests. There are other unit/mock e2e tests that do have values for the lifetime field, but that's by inputting it during the test, not actually from the JSON.